test/htmlparser.spec.js

```diff
@@ -1,36 +1,36 @@
-const expect = require('expect.js')
-
 const HTMLParser = require('../dist/htmlhint.js').HTMLParser
 
-expect.Assertion.prototype.event = function (type, attr) {
-  const obj = this.obj
-
-  if (attr !== undefined) {
-    attr.type = type
-  } else {
-    if (typeof attr === 'string') {
-      attr = { type: type }
+expect.extend({
+  toEvent(received, type, attr) {
+    if (attr !== undefined) {
+      attr.type = type
     } else {
-      attr = type
+      if (typeof attr === 'string') {
+        attr = { type: type }
+      } else {
+        attr = type
+      }
     }
-  }
-  this.assert(
-    eqlEvent(obj, attr),
-    () =>
-      `expected "${JSON.stringify(obj)}" to event "${JSON.stringify(attr)}"`,
-    () =>
-      `expected "${JSON.stringify(obj)}" not to event "${JSON.stringify(attr)}"`
-  )
-}
-
-function eqlEvent(event, attr) {
-  for (const name in attr) {
-    if (name !== 'attrs' && event[name] !== attr[name]) {
-      return false
+    for (const name in attr) {
+      if (name !== 'attrs' && received[name] !== attr[name]) {
```

Comment:
OK, fixed the first issue of swallowing everything, but I'm not sure how to write a failing tests for this to really test it out